### ISSUE TITLE
Torque errors on upload

### ISSUE THREAD

**Issue by slifty** [OpenTechStrategies/torque#64]
When I run ETL I'm seeing a few hundred errors in my torque logs similar to:

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/208884/140090394-727c1f31-62ff-4ea6-819f-d078bd95b42c.png">

(one error per upload)

**Comment by slifty** [comment]
I don't know if this is related, but I also see a ton of messages like this:

<img width="1077" alt="image" src="https://user-images.githubusercontent.com/208884/140092118-ca0edfdd-950e-439a-9aaf-59d15a3e440b.png">

**Comment by frankduncan** [comment]
The first message is weird.  What are the parameters that are being passed in?  Some debugging will help.

The second message is mediawiki attempting to render the page when you do something to it (like edit it via the etl pipeline that creates all the pages).  I have waffled between keeping this error message around because it's the same one (and very useful) if something is messed up with a users permissions, and removing it because anytime we upload, mediawiki tries and renders the page (for search caching, among other things) with no user group.

**Comment by slifty** [comment]
@frankduncan got it -- will debug around the first.

Re the second, I think the right thing here would be to figure out a way to have that message NOT appear when the system is behaving as expected (e.g. during ETL) but still appear when the system is not behaving.  Is there a way to modify ETL to make sure the proper information is passed, or some other indicator that would prevent the warning from triggering?

**Comment by frankduncan** [comment]
@slifty Yeah, we coudl disable the #tdcrender tag when the user is on a specific block list, and we could do things like add the bots we use for ETL to that.  That might be good enough.  The only thing we'd need is a global variable in torque that says "hey, don't render when this is set" and let script set it.

**Comment by frankduncan** [comment]
The first is actually caused a bug I introduced when uplaoding new attachments.  Pushing a fix to main now (as it also broke production)
